strax/processing/peak_splitting.py

```diff
@@ -6,7 +6,7 @@
 
 
 @export
-def split_peaks(peaks, records, to_pe, algorithm='local_minimum',
+def split_peaks(peaks, records, to_pe, n_top_pmts, algorithm='local_minimum',
```

Comment:
Also here, please update the doc string.

Reply:
Check